src/embedded/journey.test.ts

```diff
@@ -33,10 +33,18 @@ describe('embedded', () => {
     expect(opt.env).toHaveProperty('ENABLE_MODULES', 'text2vec-contextionary');
     expect(opt.env).toHaveProperty('CONTEXTIONARY_URL', 'contextionary:9999');
     expect(opt.env).toHaveProperty('QUERY_DEFAULTS_LIMIT', 100);
+    expect(opt.env).toHaveProperty('CLUSTER_HOSTNAME', 'Embedded_at_7777');
     expect(opt.host).toEqual('somehost.com');
     expect(opt.port).toEqual(7777);
   });
 
+  it('overrides default env vars with inherited exported ones', () => {
+    process.env.CLUSTER_HOSTNAME = 'custom-hostname';
+    const opt = new EmbeddedOptions();
+    // eslint-disable-next-line prettier/prettier
```

Comment:
Any reason why we can't format the next line in a prettier-conforming way?

Reply:
Hmm I don't remember what the original problem was, I just removed this line and there are no linter complaints